steps/azure-disks-step-disk-delete/step.py

```diff
@@ -37,7 +40,7 @@
   resource_group_name = resource_id.split('/')[4] # Resource group name
   disk_name = resource_id.split('/')[8] # VM name
   print('Deleting Azure Disk {0} in Resource Group {1}'.format(disk_name, resource_group_name))  
-  async_disk_operation = compute_client.disks.delete(resource_group_name, disk_name)
+  async_disk_operation = compute_client.disks.begin_delete(resource_group_name, disk_name)
```

Comment:
Are you sure the return type of this method is the same as the old one?

Reply:
Good question. Yup: https://docs.microsoft.com/en-us/python/api/azure-mgmt-compute/azure.mgmt.compute.v2017_03_30.operations.disksoperations?view=azure-python#begin-delete-resource-group-name--disk-name----kwargs-